### ISSUE TITLE
Add images to the output component

### ISSUE THREAD

**Issue by anshumanv** [anshumanv/evolutions-webapp#27]
Use pokemon.num to access pokemon's ID and retrieve pokemon sprite/svg etc with that ID.

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [ ] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.
